package.json

```diff
@@ -46,12 +47,11 @@
     "gulp-sourcemaps": "^2.2.0",
     "gulp-uglify": "^2.0.1",
     "gulp-useref": "^3.1.2",
-    "gulp-util": "^3.0.7",
     "http-proxy-middleware": "^0.17.3",
     "jasmine": "^2.4.1",
     "karma": "^2.0.2",
```

Comment:
The dependencies of karma v2.0.x contain vulnerabilities. Upgrading to v3.0.0 will require migration effort.